src/transport/SecureSessionMgr.h

```diff
@@ -94,13 +94,19 @@ class EncryptedPacketBufferHandle final : private System::PacketBufferHandle
     CHIP_ERROR InsertPacketHeader(const PacketHeader & aPacketHeader) { return aPacketHeader.EncodeBeforeData(*this); }
 #endif // CHIP_ENABLE_TEST_ENCRYPTED_BUFFER_API
 
+#ifdef CHIP_CONFIG_TEST
     void operator=(PacketBufferHandle && aBuffer) { PacketBufferHandle::operator=(std::move(aBuffer)); }
+#endif
```

Comment:
```suggestion
#ifdef CHIP_CONFIG_TEST
  public:
#else
  private:
#endif // CHIP_CONFIG_TEST
    void operator=(PacketBufferHandle && aBuffer) { PacketBufferHandle::operator=(std::move(aBuffer)); }
```

is another option.  It ensures that we don't have different impls for tests and non-tests...